release/release_tests.yaml

```diff
@@ -1258,38 +1255,19 @@
     wait_for_nodes:
       num_nodes: 4
 
+  variations:
```

Comment:
You probably have to add this to the schema.json

Reply:
You spot on on whether we should have a separated schema validation for TestDefinition. I don't have an answer that I can feel happy with just yet, so there are a lot of validation at runtime during the parsing of TestDefinition for now.

Cloning schema.json into two similar files are definitely not good (for maintenance, etc.), so I need inheritance in json, which made me want to defer this problem for another diff.